### PR TITLE
x11idle: update version, repository

### PR DESCRIPTION
**Pull request by swflint**
###### Motivation for this change

Org Mode has moved its main repository, therefore this package should be updated.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by gebner** [comment]
@GrahamcOfBorg build x11idle

**Comment by swflint** [comment]
@GrahamcOfBorg build x11idle

**Comment by matthewbauer** [comment]
@GrahamcOfBorg build x11idle

**Review by matthewbauer — CHANGES_REQUESTED** [review]
Wrong hash

**Comment by matthewbauer** [comment]
@GrahamcOfBorg build x11idle

File: pkgs/tools/misc/x11idle/default.nix [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
-stdenv.mkDerivation {
-  name = "x11idle-unstable-2018-06-16";
+stdenv.mkDerivation rec {
+
+  version = "9.1.9";
+  name = "x11idle-org-${version}";
 
   src = fetchgit {
     url = "https://code.orgmode.org/bzg/org-mode.git";
```
Comment (matthewbauer) [inline pkgs/tools/misc/x11idle/default.nix]: Would it be possible to jut use https://code.orgmode.org/bzg/org-mode/raw/release_9.1.13/contrib/scripts/x11idle.c as the source? That would be much smaller.

**Comment by chreekat** [comment]
Ping. @swflint is this PR still worth working on for you?

**Comment by swflint** [comment]
Yes.

**Comment by swflint** [comment]
@chreekat @matthewbauer Please review as available.

File: pkgs/tools/misc/x11idle/default.nix [hunk 2]
```diff
@@ -1,12 +1,13 @@
-{ stdenv, fetchgit, libXScrnSaver, libX11 }:
+{ stdenv, xlibs, fetchurl, libXScrnSaver, libX11 }:
```
Comment (Enteee) [inline pkgs/tools/misc/x11idle/default.nix]: remove unneeded `xlibs`:

```suggestion
{ stdenv, fetchurl, libXScrnSaver, libX11 }:
```

this should also make the build pass.

**Review by Enteee — CHANGES_REQUESTED** [review]

**Review by Enteee — APPROVED** [review]

**Pull request merged**